ngrinder-controller/src/main/java/org/ngrinder/starter/NGrinderControllerStarter.java

```diff
@@ -243,6 +245,23 @@ public static void main(String[] args) {
 		SpringApplication.run(NGrinderControllerStarter.class, args);
 	}
 
+	private static void checkTmpDirProperty() {
+		String userHome = System.getProperty("user.home");
+		String ngrinderHome = Config.getUserHome();
+		String javaTmpDir = System.getProperty("java.io.tmpdir");
+
+		if (!javaTmpDir.startsWith(userHome) && !javaTmpDir.startsWith(ngrinderHome)) {
```

Comment:
I think this should be flexiable..
I think you can check both dir equality only.